lib/player.js

```diff
@@ -416,8 +417,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
    *    will remain detached.
    * @param {function(shaka.Player)=} dependencyInjector Optional callback
    *   which is called to inject mocks into the Player.  Used for testing.
+   * @param {?HTMLCanvasElement} [canvas] Optional canvas where LCEVC
+   *    video will be drawn.
+   * @param {?Object} [lcevcDilConfig] Optional canvas where LCEVC
+   *    video will be drawn.
```

Comment:
Let's create setters for these instead.

Reply:
Can you please explain what is expected here. Thanks.